websites/N/NationStates/presence.ts

```diff
@@ -0,0 +1,359 @@
+const images = {
+		header: "https://i.imgur.com/ffxJj5E.png",
+		logo: "https://i.imgur.com/04gnehi.png",
+		envelope: "https://i.imgur.com/6Hp3f7k.png",
+		flag: "https://i.imgur.com/BAzbs0d.png",
+		forum: "https://i.imgur.com/uDIj1ta.png",
+		gift: "https://i.imgur.com/28eyl2A.png",
+		globe: "https://i.imgur.com/hE7TN9d.png",
+		page: "https://i.imgur.com/pArUzoy.png",
+		person: "https://i.imgur.com/8pTlzJw.png",
+		worldassembly: "https://i.imgur.com/A09MAL2.png",
+		target: "https://i.imgur.com/nksi1kH.png",
+	},
```

Comment:
Can do, but the review came a bit later than expected and I have left for vacation since then. 
I won't be near a computer this week, so I ask of you not to lock/delete this PR for inactivity – I will edit the code when I'll get back home next week.

Thanks.

Reply:
Set it as a draft